### ISSUE TITLE
Tag 8.6.5 not found in repository sameersbn/gitlab

### ISSUE THREAD

**Issue by nathanleclaire** [sameersbn/docker-gitlab#666]
```
Tag 8.6.5 not found in repository docker.io/sameersbn/gitlab
```

Just FYI: When attempting to `docker-compose up` on `master`, I ran into this issue.  Indeed, it's not present at https://hub.docker.com/r/sameersbn/gitlab/tags/ -- should it be `sameersbn/gitlab:8.6.4`?

cc @sameersbn

**Comment by lenovouser** [comment]
Duplicate of #662

**Comment by nathanleclaire** [comment]
Why have the Compose file set to an image that doesn't exist though?  Even if the project is blocked on upgrading to that version, couldn't an older one still be used?

**Comment by sameersbn** [comment]
@nathanleclaire I was careless while releasing 8.6.5. The changes from `8.6.4` to `8.6.5` were insignificant enough that and I went ahead and tagged a new release before trying to build the image locally. This was a big mistake from my side.

As it stands today, I have worked around the issue and made new releases. Sorry for the inconvenience.

**Comment by nathanleclaire** [comment]
Cool no problem thanks @sameersbn
